proto/file.proto

```diff
@@ -20,6 +20,8 @@ service FileService {
 message CreateUploadRequest {
     string bucket = 1;
     string name = 2;
+    string ownerID = 4;
+    string parent = 5;
```

Comment:
There are messages that have an `appID` field which is not defined in any of the schemas and is not used anywhere in the file-service's code. Remove it and compile the protobuf again.

Reply:
Also remove `Bucket` field from `GenerateKeyRequest`